pkg/oauth/server/osinserver/registrystorage/storage.go

```diff
@@ -84,6 +84,13 @@ func (w *clientWrapper) GetUserData() interface{} {
 	return w.client
 }
 
+func (w *clientWrapper) GetExpiration(defaultExpiration int32) int32 {
+	if w.client.AccessTokenMaxAgeSeconds != nil {
```

Comment:
Should there be an upper bound beyond which users can't specify?  Under admin control?  Or can we convincingly argue that only admins should be able to edit OAuthClients?